server/src/controllers/server.controller.ts

```diff
@@ -109,7 +109,7 @@ export class ServerController {
   }
 
   @Get('version-check')
-  @Authenticated()
+  @Authenticated({ permission: Permission.ServerVersionCheck, admin: true })
```

Comment:
Please remove the admin: true, since this isn't an admin only route.

Reply:
I would have assumed it is, but obviously not :see_no_evil: